src/transformers/models/vipllava/modeling_vipllava.py

```diff
@@ -282,6 +282,21 @@ def resize_token_embeddings(self, new_num_tokens: Optional[int] = None, pad_to_m
         self.vocab_size = model_embeds.num_embeddings
         return model_embeds
 
+    def get_image_features(
```

Comment:
@zucchini-nlp `make fix-copies` copied this over but looking at the forward pass of vipllava, the image features selection is slightly different since they select features from for the layers -2, -5, -8, -11 and 6
I don't think it's breaking anything since this function is not called. Note that we'll need ` vision_feature_layers: list[int]` and not only a single one!

Reply:
Hmm, in that case better to make the method accept list in vipllava and add a "#Ignore copy" statement so that it is not copied from llava. That way we can use the `get_image_features` directly

I don't think it's a good idea to merge smth which doesn't work, even tho it technically doesn't break anything

Reply:
agreed 💯 fixed now in https://github.com/huggingface/transformers/pull/33696/commits/f75e94470fd8d88b0e8e702826728660a9e618a9